### DISCUSSION TITLE
Generation not coming out right

### DISCUSSION THREAD

**Discussion by Drewbears** [shun126/UE5-DungeonGeneratorDemo#26] [General]
![Capture](https://github.com/shun126/UE5-DungeonGeneratorDemo/assets/88593669/4e844368-5a96-4f59-99df-a5daa5e5daa3)

Im not sure whats happening or what i did wrong but i could use some help

**Comment by shun126** [comment]
I would like to check the situation.Is the problem that the mesh is falling apart?

https://github.com/shun126/UE5-DungeonGeneratorDemo/releases

The assets in the sample project might be helpful.

https://github.com/shun126/UE5-DungeonGeneratorDemo/wiki/enus_Prepare-parts

Or visit the Wiki to check the mesh.

**Reply by Drewbears** [reply]
Still having issues with it not working properly

**Reply by shun126** [reply]
What type of problem is it?
I think the problem is that each part is separate from the image.
